lib/SimpleSAML/Metadata/MetaDataStorageSource.php

```diff
@@ -53,7 +53,7 @@ public static function parseSources($sourcesConfig)
      *
      * @param array $sourceConfig Associative array with the configuration for this metadata source.
      *
-     * @return mixed An instance of a metadata source with the given configuration.
+     * @return MetaDataStorageSource An instance of a metadata source with the given configuration.
```

Comment:
I'd like to see the fully namespaced class name in phpdocs

Reply:
Fixed